publish.js

```diff
@@ -115,21 +118,21 @@ function buildMetadata (spec, auth, registry, manifest, tardata, opts) {
     readme: manifest.readme || ''
```

Comment:
We _might_ be able to stop doing this, since we only serve the readme that's in the actual tarball nowadays anyway.  Would make publishes go faster.